src/main/java/project/mybookshop/validation/Email.java

```diff
@@ -0,0 +1,17 @@
+package project.mybookshop.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Email {
+    String message() default "Invalid format email";
+    Class<?>[] groups() default {};
+    Class<? extends Payload>[] payload() default {};
+}
```

Comment:
@Email already exist. 
Reuse existing solutions to decrease code amount + it is already tested solutions.